pkgs/development/libraries/libdeflate/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "libdeflate-${version}";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "ebiggers";
+    repo = "libdeflate";
+    rev = "v${version}";
+    sha256 = "1wqxwza6rwmhrsy9sw86pdcd0w742gbzsy9qxnq6kk59m6h1dbsb";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile --replace /usr $out
```

Comment:
I have asked upstream to make `/usr` configurable in https://github.com/ebiggers/libdeflate/pull/38